src/main/resources/portal.properties.EXAMPLE

```diff
@@ -144,6 +144,7 @@ show.oncokb=true
 
 # Enable Chang's hotspot list (true, false)
 show.hotspot=true
+hotspots.url=http://cancerhotspots.org/api/hotspots/single
```

Comment:
Isn't this supposed to be `http://cancerhotspots.org/api/` as opposed to `http://cancerhotspots.org/api/hotspots/single`?
Seems like we are already appending `hotspots/single/` in the `ProxyController` class?